### PR TITLE
Add pull_policy in compose files

### LINKED ISSUE #368: Startup Script not working on specific docker compose versions
## Short
On some docker compose distributions the --pull=always flag is not implemented

## Proposal 
On some docker compose distributions the --pull=always flag is not implemented causing the startup scripts to fail. 
We need to check the docker compose availability and make usre to use the pull_always directly in the compose file rather than the docker compose command unfortunately.

## Solution 
- add docker compose installation inside the script
- add `pull_policy=always` in compose files

## Status
development 

## Checklist
- [ ] Discussed
- [x] Solved
- [x] Tested

### PR DESCRIPTION
**Pull request by giobart**
fixes #368 

- removed -pull=always from docker compose command 
- added pull_policy in compose files 
- removed obsolete compose files
- added docker compose plugin auto-install

### CODE REVIEW

File: run-a-cluster/1-DOC.yaml
Comment (Mjaethers) [inline run-a-cluster/1-DOC.yaml:1]: Was not applying the pull policy to mongo and grafana deliberate?

**Review by Mjaethers — COMMENTED** [review]
Looks just one thing reagrding the 1-DOC compose file

File: run-a-cluster/1-DOC.yaml
Comment (giobart) [reply run-a-cluster/1-DOC.yaml:1]: absolutely not thanks for noticing! Fixed

**Review by Mjaethers — APPROVED** [review]
Looks good to me!

**Pull request merged**